### ISSUE TITLE
What does a high negative z-score indicates?

### ISSUE THREAD

**Issue by GreeshmaThulasi** [rstraver/wisecondor#47]
Hi,
@rstraver,
I'm getting a high negative z-score of -54.3 in chr5 for a sample of pregnant women.
[NA-349plot_z.pdf](https://github.com/VUmcCGP/wisecondor/files/3684597/NA-349plot_z.pdf)
Is that mean a deletion over a region in chr5? 
Please help me out.

Thanks&Regards,
Greeshma Thulasi

**Comment by rstraver** [comment]
That would indeed indicate a deletion, however, assuming you are working with pregnancy data it does not necessarily mean it's fetal or maternal right away as it's lengthy and probably fairly subtle (could be mosaicism in the mother as well).

**Comment by GreeshmaThulasi** [comment]
Thank you @rstraver for the reply.

I've read that, the green color in z-score plot is indicating that, those are reads from mother and the orange color indicates it's fetal. we have sequenced this sample two times. Once the deletion region in the plot was orange in color and next time it was green in color.
[NA-349_plot_z.pdf](https://github.com/VUmcCGP/wisecondor/files/3689041/NA-349_plot_z.pdf)
We have done karyotyping of for mother and it came out to be normal.
Did Amniocentesis microarray for fetus also, which too came out to be normal.

What does this mean?

Thanks&Regards,
Greeshma Thulasi

**Comment by rstraver** [comment]
Well, the green means it was found to be deviating a bit much for it to be (only) fetal, suggesting it could be maternal. Alternatively it could be 'fetal' but with a rather strong copy change (i.e. loss of both alleles). I am saying 'fetal' with quotes as it's possibly only the placenta in that case, and currently I'm not 100% sure if that would be visible in the amniocentesis as it's been a while since I worked on this.

It's probably best to shoot my supervisor from that period an e-mail explaining this question:
Sistermans, E.A. (Erik) <e.sistermans@vumc.nl>

**Comment by JspSrs** [comment]
Indeed, most probable a placental mosaic. With NIPT (thus via mother’s blood) one is in fact typing the placenta, not the fetus.

A recent article by Van Opstal et al (Prenatal Diagnosis 2018, PMID30187503, also in PMC) gives insight in this matter.

And, indeed, the supervisor of Roy (Sistermans, E.A. (Erik) e.sistermans@vumc.nl<mailto:e.sistermans@vumc.nl>) can explain in more detail.

**Comment by GreeshmaThulasi** [comment]
Thank you @rstraver and @JspSrs for the reply.
It could be a case of confined fetal mosaicism, which may not pick up in amniocentesis. Right?
But the patient is continuing the pregnancy, since the invasive procedures given a negative result.

Thanks & Regards,
Greeshma Thulasi

**Comment by rstraver** [comment]
I believe so, and I suppose you can check the placenta after birth to see if this really is/was the case (I'm curious to hear the outcome).

Best of luck to you and your patient!

**Comment by GreeshmaThulasi** [comment]
Ok. Thank you.
I'll try to do the karyotyping of placenta after birth.

I'll sure let you know the outcome too.

Thanks&Regards,
Greeshma Thulasi

**Comment by Valence-H** [comment]
> Ok. Thank you.
> I'll try to do the karyotyping of placenta after birth.
> 
> I'll sure let you know the outcome too.
> 
> Thanks&Regards,
> Greeshma Thulasi

Today is 2020.6.18, 
Does the baby come out? I am curious to the result too.
Thank you.
